README.md

```diff
@@ -291,6 +291,16 @@ AuthModule.forRoot({
           scope: 'read:users',
         },
       },
+
+      // Using an URI matcher
+      {
+        uriMatcher: (uri) => uri.indexOf('/api/orders') > -1,
```

Comment:
We should explain more how `uriMatcher` works, what `uri` is, and what the developer is expected to return and why. It is pretty simple and I think a short sentence would cover all of that as an introduction.

"If you need more fined-grained control over the URI matching, you can do.... . You might want to do this in scenarios where...". etc. The scenario you gave me where you might have 10 routes that require a token but 1 of them doesn't would be perfect in here as an illustration.

Reply:
I took a slightly different approach and moved the uri matching to its own section in the readme: https://github.com/auth0/auth0-angular/pull/122/commits/695978845d8d1713c32a1815bbfae3f08cc1cfa7